src/components/marquee-wizard/marquee-wizard.component.html

```diff
@@ -37,8 +37,6 @@
 
             <li *ngFor="let step of steps; let index = index"
                 role="tab"
-                uxTabbableListItem
```

Comment:
Reorganised these properties to match the `WizardComponent` for consistency and readability.